resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Is listening to the woes of a fellow Clanmate",
+            "Notices an apprentice struggling with a task and offers {PRONOUN/m_c/poss} help",
+            "Is doing some apprentice tasks around camp, to help lighten the load",
+            "Is sharing tongues with friends",
+            "Is keeping the injured cats company",
+            "Ponders the burdens of others",
+            "Can't stop smiling because of another Clanmates good mood",
+            "Is feeling really sad, but tries not to let it show",
+            "Says sorry before {PRONOUN/m_c/subject} {VERB/m_c/bite/bites} into {PRONOUN/m_c/poss} fresh-kill",
+            "Helps solve a conflict by mediating both sides",
+            "Purrs contently at the thought of {PRONOUN/m_c/poss} Clanmates being happy"
+        ],
+        "main_trait_constraint": [
+            "compassionate"
+        ]
+    },
+    {
+        "id": "compassionate_warrior_to_living_med_cat",
+        "thoughts": [
+            "Is helping the medicine cat organize herb stores"
+        ],
+        "random_status_constraint": [
+            "medicine cat"
+        ],
+        "main_trait_constraint": [
+            "compassionate"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "competitive_gen",
+        "thoughts": [
+            "Is pestering other cats to challenges around camp",
+            "Challenged a Clanmate to a staring contest",
+            "Challenged a Clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
+            "Has been a good sport about losing to a Clanmate",
+            "Has been a bit of a bad sport about losing to a Clanmate",
+            "Challenged a Clanmate to a race",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and won",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and lost",
+            "Makes every patrol into a competition",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Watched a Clanmate do a chore {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} really good at to see how {PRONOUN/m_c/object} can improve",
+            "Is practicing doing things around camp as fast as possible, just in case"
+        ],
+        "main_trait_constraint": [
+            "competitive"
+        ]
+    },
+    {
+        "id": "confident_warrior",
+        "thoughts": [
+            "Is puffing {PRONOUN/m_c/poss} chest out",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} the smartest cat in the Clan",
```

Comment:
might be nice to see some of these "best of the clan" thoughts moved into thought groups that require the associated skill.  confident claims feel like they should come from some truth, as opposed to source-less claims an arrogant cat might make.  would additionally be cool to see corresponding thoughts of this nature added for *all* the different skill trees, however that's just a suggestion, not a requirement from me.

Reply:
will add after confirmation on my other skill related thing